adapters/parent/pom.xml

```diff
@@ -248,5 +258,131 @@
         </dependency>
       </dependencies>
     </profile>
+    
+    <profile>
+      <id>read-ports</id>         
+      <activation>
+        <property>
+          <name>read-ports</name>
+        </property>
+      </activation>
+      <properties>
+        <docker.dir>${project.basedir}/../../tests/target/docker</docker.dir>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>properties-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>read</id>
+                <phase>initialize</phase>
+                <goals>
+                  <goal>read-project-properties</goal>
+                </goals>
+                <configuration>
+                  <quiet>true</quiet>
+                  <files>
+                    <file>${docker.dir}/jaeger.port.properties</file>
+                    <file>${docker.dir}/infinispan.port.properties</file>
+                    <file>${docker.dir}/broker.port.properties</file>
+                    <file>${docker.dir}/jdbc-db.port.properties</file>
+                    <file>${docker.dir}/mongodb.port.properties</file>
+                    <file>${docker.dir}/qpid.port.properties</file>
+                    <file>${docker.dir}/kafka.port.properties</file>
+                    <file>${docker.dir}/auth.port.properties</file>
+                    <file>${docker.dir}/deviceregistry.port.properties</file>
+                    <file>${docker.dir}/commandrouter.port.properties</file>
+                  </files>
+                </configuration>
+              </execution>
+              <!-- 
+                set system properties adding ports as system
+                properties - so they could be resolved in .env 
+              -->
```

Comment:
do we still need this section?

Reply:
Unfortunately, with the new model it seems the `read-props` profile doesn't work at all. 
It seems that `quarkus.application-dev.properties` is resolved before reading properties and with or without setting mongodb.port / kafka.port as system properties by `read-props` the fixed (not the read) ports are used.
I think that maybe I should drop support for random ports for local services. Having option to start services locally with fixed (but configurable) ports should be sufficient to fulfill the main goal - to provide support for local test/debug of hono.
Support for random ports would greatly complicate that feature without big value. It may be left for later if requested. 
What do you think?

Reply:
I agree. Let's start with fixed ports only for now.